docs/sdk/push-notifications/fragments/ios/getting-started.md

```diff
@@ -37,16 +44,16 @@ Use the following steps to connect push notification backend services to your ap
 
 1. The `Podfile` that you configure to install the AWS Mobile SDK must contain the `AWSPinpoint` pod:
```

Comment:
Weird line. Is `AWSMobileClient` not a "must contain?"

Reply:
Weird indeed. I made it simpler, let me know what you think.

Reply:
LGTM!